purchases/src/main/kotlin/com/revenuecat/purchases/models/Period.kt

```diff
@@ -41,6 +46,18 @@ data class Period(
         YEAR,
         UNKNOWN,
     }
+
+    val valueInMonths: Double
```

Comment:
I guess detekt doesn't enforce it, but it would be nice to always add docstrings to public methods.

Reply:
Completely agree, there are actually rules to check for that but we don't have them enabled: https://detekt.dev/docs/rules/comments/#undocumentedpublicproperty. Create a linear ticket to track this.